src/Main.hs

```diff
@@ -113,6 +117,18 @@ main = do
                           )
             mapM_ handleResults results
 
+      (Coq f) -> do
+        contents <- readFile f
+        case parse (lexer contents) of
```

Comment:
I made a continuation passing style `proceed` function so we can avoid all of these nested cases

Reply:
fixed in https://github.com/ethereum/act/pull/44/commits/1c2f3209d39d211b8337bbf713e13fefb0ae8652